modules/dynamic-plugins/proc-enable-plugins-rhdh-container-image.adoc

```diff
@@ -0,0 +1,34 @@
+[id="proc-enable-plugins-rhdh-container-image_{context}"]
+= Enabling plugins added in the {product-very-short} container image
+
+In the {product-very-short} container image, a set of dynamic plugins is preloaded to enhance functionality. However, due to mandatory configuration requirements, most of the plugins are disabled.
```

Comment:
👍 This is a great explanation, it clearly explains what is going on with plugins